### PR TITLE
feat: add/edit book activity be able to delete cover image

### PR DESCRIPTION
**Pull request by c25vdw**

### CODE REVIEW

File: app/src/main/res/values/arrays.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <array name="image_actions_new">
+        <item>Upload new image</item>
+        <item>Cancel</item>
+    </array>
+    <array name="image_actions_exist">
+        <item>Replace with new image</item>
+        <item>Delete cover image</item>
+        <item>Cancel</item>
```
Comment (zhpeng811) [inline app/src/main/res/values/arrays.xml:10]: I wonder if these can use string resource? Since this is something that will be displayed to the user

**Review by zhpeng811 — APPROVED** [review]
Looks like string resource in arrays.xml can be done simply like this:

image

your call if you want to change it, I'm good with both

other than that it lgtm

File: app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/OwnedListAdapter.java [hunk 2]
```diff
@@ -47,7 +39,17 @@ public OwnedListAdapter(List<Book> books, onItemClickListener itemClickListener,
         View itemView = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_book_list, parent, false);
         return new OwnedListAdapter.ViewHolder(itemView, itemClickListener, deleteListener, viewRequestsListener);
     }
+    public interface onViewRequestsListener {
```
Comment (kdvuong) [inline app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/OwnedListAdapter.java:42]: capitalize interface name?

File: app/src/main/java/com/cmput301f20t21/bookfriends/ui/add/AddEditViewModel.java [hunk 3]
```diff
@@ -43,22 +60,54 @@ public AddEditViewModel(IAuthRepository authRepository, IBookRepository bookRepo
...
+
+    public Book getOldBook() {
+        return oldBook;
+    }
+
+    public LiveData<Uri> getLocalImageUri() {
+        return localImageUri;
+    }
+
+    public void setLocalImageUri(Uri uri) {
```
Comment (kdvuong) [inline app/src/main/java/com/cmput301f20t21/bookfriends/ui/add/AddEditViewModel.java:80]: @Nullable here?

**Review by kdvuong — APPROVED** [review]
lgtm, a few small comments.

**Pull request merged**